src/js/api/config.js

```diff
@@ -50,7 +50,8 @@ const Defaults = {
         unmute: 'Unmute'
     },
     renderCaptionsNatively: false,
-    nextUpDisplay: true
+    nextUpDisplay: true,
+    itemReady: false
```

Comment:
This goes in `INITIAL_PLAYER_STATE`, not here in the default config (it's not configurable).